src/Symfony/Component/HttpKernel/Controller/ArgumentValueResolverInterface.php

```diff
@@ -18,6 +18,8 @@
  * Responsible for resolving the value of an argument based on its metadata.
  *
  * @author Iltar van der Berg <kjarli@gmail.com>
```

Comment:
Maybe a bit late cr seeing the code is already merged but shouldn't we have some sort off:
```
trigger_deprecation('symfony/http-kernel', '6.2', 'The "%s" interface is deprecated.', ValueResolverInterface::class);
```
to inform the user on code execution @nicolas-grekas ?

Reply:
DebugClassLoader will trigger it when you implement it, which is much better than triggering it when it is autoloaded (core classes still need to implement it for BC)

Reply:
Cool is this the case for all interfaces that are deprecated when you are running debug mode?